<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_window</class>
 <widget class="QWidget" name="Login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>莫狗健身</string>
  </property>
  <property name="windowIcon">
   <iconset resource="login_window_res.qrc">
    <normaloff>:/window/icons/dog.svg</normaloff>:/window/icons/dog.svg</iconset>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="login_page">
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>370</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(107,152,246);
	border-radius:20px;
	border:none;
	color: rgb(255, 255, 255);
	outline:0px;
}</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>320</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/window/icons/lock.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="toRegister_Button">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>420</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border:0px;
	color: rgb(56, 92, 188);
}</string>
     </property>
     <property name="text">
      <string>立即注册？</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>540</width>
       <height>342</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:./window/images/login_bg.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="UsernameLineEdit">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>253</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: rgb(243, 245, 253);
	border-radius:20px;
	border:none;
	padding-left: 38px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>用户名</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/window/icons/user.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border: none;
	border-top-right-radius:10px;
	border-bottom-left-radius:10px;
	outline:0px;
}

QPushButton:hover{
	background-color: rgb(250,76,62);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="login_window_res.qrc">
       <normaloff>:/window/icons/close.svg</normaloff>:/window/icons/close.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="rememberPwd_checkBox">
     <property name="geometry">
      <rect>
       <x>685</x>
       <y>419</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
	
}
QCheckBox::indicator{/*选择框尺寸*/
    background-color: rgb(255, 255, 255);
    border: 1px solid #385CBC;
	width: 11px;
	height: 11px;
	border-radius:2px;
}

QCheckBox::indicator:checked {
	image: url(:/window/icons/check.svg);
}
</string>
     </property>
     <property name="text">
      <string>记住密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>450</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 70, 73);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>38</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/window/icons/bone.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="PwdLineEdit">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>313</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: rgb(243, 245, 253);
	border-radius:20px;
	border:none;
	padding-left: 38px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(56, 92, 188);</string>
     </property>
     <property name="text">
      <string>欢迎使用莫狗AI智能健身系统 :)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>971</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>110</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>莫狗健身</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>715</x>
       <y>180</y>
       <width>148</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>不做细狗，从我做起</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>120</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/window/icons/js_l.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>120</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/window/icons/js_r.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>LoginButton</zorder>
    <zorder>toRegister_Button</zorder>
    <zorder>label_2</zorder>
    <zorder>UsernameLineEdit</zorder>
    <zorder>label_7</zorder>
    <zorder>closeButton</zorder>
    <zorder>rememberPwd_checkBox</zorder>
    <zorder>label_10</zorder>
    <zorder>PwdLineEdit</zorder>
    <zorder>label_9</zorder>
    <zorder>label_8</zorder>
    <zorder>label_3</zorder>
    <zorder>label_6</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>errorLabel</zorder>
   </widget>
   <widget class="QWidget" name="register_page">
    <widget class="QLabel" name="label_11">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>971</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="toLogin_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border: none;
	border-top-left-radius:10px;
	border-bottom-right-radius:10px;
	outline:0px;
}

QPushButton:hover{
	background-color: #D6D6D6;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="login_window_res.qrc">
       <normaloff>:/window/icons/back.svg</normaloff>:/window/icons/back.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>529</x>
       <y>150</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/window/images/approve request.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>380</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{
	border-top: 10px solid qlineargradient(y0:0, y1:1,stop: 0 #ececef, stop: 1 white);
	border-left: 10px solid qlineargradient(x0:0, x1:1,stop: 0 #ececef, stop: 1 white);
	border-bottom: 10px solid qlineargradient(y0:0, y1:1,stop: 0 white, stop: 1  #ececef);
	border-right: 10px solid qlineargradient(x0:0, x1:1,stop:  0 white, stop: 1 #ececef);
}

QLineEdit{
	background-color: rgb(243, 245, 253);
	border-radius:10px;
	border:none;
	padding-left: 15px;
}

QGroupBox{
	background-color: rgb(243, 245, 253);
	border-radius:10px;
	border:none;
	font-size: 15px;
	padding-left: 15px;
}</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(56, 92, 188);</string>
      </property>
      <property name="text">
       <string>注册莫狗健身 :)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registerButton">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>360</y>
        <width>270</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(253, 183, 84);
	border-radius:20px;
	border:none;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerUsernameLineEdit">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>90</y>
        <width>270</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>用户名</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerPasswordLineEdit">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>140</y>
        <width>270</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerNicknameLineEdit">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>190</y>
        <width>270</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>昵称</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerHeightLineEdit">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>290</y>
        <width>130</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>身高(厘米)</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerWeightLineEdit">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>290</y>
        <width>130</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>体重(斤)</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="registerErrorLabel">
      <property name="geometry">
       <rect>
        <x>89</x>
        <y>410</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 70, 73);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>240</y>
        <width>270</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox{
padding:0px;
}

QPushButton{
	background-color: transparent;
}
QPushButton::checked {
	background-color: rgb(85, 175, 252);
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="registerManPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>41</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
         </property>
         <property name="text">
          <string>男</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="registerWomanPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>41</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
         </property>
         <property name="text">
          <string>女</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>UsernameLineEdit</tabstop>
  <tabstop>PwdLineEdit</tabstop>
  <tabstop>LoginButton</tabstop>
  <tabstop>rememberPwd_checkBox</tabstop>
  <tabstop>toRegister_Button</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>toLogin_Button</tabstop>
  <tabstop>registerButton</tabstop>
  <tabstop>registerUsernameLineEdit</tabstop>
  <tabstop>registerPasswordLineEdit</tabstop>
  <tabstop>registerNicknameLineEdit</tabstop>
  <tabstop>registerHeightLineEdit</tabstop>
  <tabstop>registerWeightLineEdit</tabstop>
  <tabstop>registerManPushButton</tabstop>
  <tabstop>registerWomanPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="login_window_res.qrc"/>
 </resources>
 <connections/>
</ui>
